bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -113,6 +130,10 @@ pub mod pallet {
 		InvalidUpgradeParameters,
 	}
 
+	/// Relayer reward tips that were added but the nonce was already consumed.
+	#[pallet::storage]
+	pub type LostTips<T: Config> = StorageMap<_, Twox64Concat, AccountIdOf<T>, u128, ValueQuery>;
```

Comment:
Needs a stronger hasher than Twox64Concat

Reply:
Used `Blake2_256` instead. https://github.com/vgeddes/polkadot-sdk/commit/a0fda22f8e4a362f46b101909969750fd2cd6ac7

Reply:
I believe you need to use `Blake2_128Concat`. 

See the following for reasons: https://polkadot.study/tutorials/substrate-in-bits/docs/Blake2-vs-Twox64#blake2_128concat

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/23/commits/d1a5ab47740706370c520b8a237c37580a32a05e